src/main/java/net/dv8tion/jda/api/events/guild/invite/GenericGuildInviteEvent.java

```diff
@@ -27,6 +27,10 @@
  * <br>Every GuildInviteEvent is derived from this event and can be casted.
  *
  * <p>Can be used to detect any GuildInviteEvent.
+ *
+ * <h2>Requirements</h2>
+ *
+ * <p>These events require the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_INVITES GUILD_INVITES} intent to be enabled
```

Comment:
This is speculation, but I think these also require the permission to manage invites.

Reply:
I want to be sure

Reply:
@caneleex Proved this to be true